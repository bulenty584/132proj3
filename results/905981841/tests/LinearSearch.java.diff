1,3c1,15
< --- Error (Explicit) ---
< Error: v95 = v93 * v96
< Wrong argument type in Multiply
---
> 10
> 11
> 12
> 13
> 14
> 15
> 16
> 17
> 18
> 9999
> 0
> 1
> 1
> 0
> 55
