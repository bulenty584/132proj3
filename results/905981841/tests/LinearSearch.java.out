--- Error (Explicit) ---
Error: v95 = v93 * v96
Wrong argument type in Multiply
